docs/source/whatsnew/1.0.2.rst

```diff
@@ -42,6 +42,8 @@ Fixed
   validating event forecast data. (:issue:`660`, :pull:`661`)
 * Changed GFS fetch directory for compatibility with March 22, 2021, 12Z
   model upgrade. (:issue:`662`, :pull:`663`)
+* Fixed ``CLEARSKY`` GHI validation to work around NaNs in the data
+  instead of not calculating the flag at all (:pull:`673`)
```

Comment:
Shouldn't we say something like "work around data gaps that cause uneven frequencies"? The NaNs themselves are not a problem. Same comment applies to the PR title and thus the eventual commit message